plenum/test/view_change/test_view_not_changed_when_primary_disconnected_from_less_than_quorum.py

```diff
@@ -0,0 +1,6 @@
+def test_view_not_change_when_primary_disconnected_from_less_than_quorum():
```

Comment:
We need to follow TDD! How is it possible that we have TODO test while having the functionality already implemented?